pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -325,10 +326,16 @@ func changefeedPlanHook(
 		}
 	}
 
+	var e *kvpb.BatchTimestampBeforeGCError
```

Comment:
Move the var declaration to right before the if statement (reduce scope whenever possible + keep declarations as close as reasonably possible to their usages)